packages/client/ui/react-ui/src/twind.config.ts

```diff
@@ -4,18 +4,17 @@ import presetTailwind from "@twind/preset-tailwind";
 export default defineConfig({
     presets: [presetTailwind()],
     theme: {
-        screens: {
-            xs: "480px",
```

Comment:
adding this screen size was causing the client app (demo) to have broken widths. removing fixes it.